system/include/compat/avxintrin.h

```diff
@@ -44,215 +535,1271 @@ _mm_broadcast_ss(const float *__mem_addr)
+      __ret = _mm_cmpeq_ss((__a), (__b));                                      \
+    if ((__imm) == _CMP_EQ_UQ || (__imm) == _CMP_EQ_US)                        \
+      __ret = _mm_move_ss(                                                     \
+        (__a),                                                                 \
+        _mm_or_ps(_mm_cmpeq_ss((__a), (__b)), _mm_cmpunord_ss((__a), (__b)))); \
+    if ((__imm) == _CMP_LT_OS || (__imm) == _CMP_LT_OQ)                        \
+      __ret = _mm_cmplt_ss((__a), (__b));                                      \
+    if ((__imm) == _CMP_LE_OS || (__imm) == _CMP_LE_OQ)                        \
+      __ret = _mm_cmple_ss((__a), (__b));                                      \
+    if ((__imm) == _CMP_UNORD_Q || (__imm) == _CMP_UNORD_S)                    \
+      __ret = _mm_cmpunord_ss((__a), (__b));                                   \
+    if ((__imm) == _CMP_NEQ_UQ || (__imm) == _CMP_NEQ_US)                      \
+      __ret = _mm_cmpneq_ss((__a), (__b));                                     \
+    if ((__imm) == _CMP_NEQ_OQ || (__imm) == _CMP_NEQ_OS)                      \
+      __ret = _mm_move_ss((__a),                                               \
+                          _mm_andnot_ps(_mm_cmpunord_ss((__a), (__b)),         \
+                                        _mm_cmpneq_ss((__a), (__b))));         \
+    if ((__imm) == _CMP_NLT_US || (__imm) == _CMP_NLT_UQ)                      \
+      __ret = _mm_cmpnlt_ss((__a), (__b));                                     \
+    if ((__imm) == _CMP_ORD_Q || (__imm) == _CMP_ORD_S)                        \
+      __ret = _mm_cmpord_ss((__a), (__b));                                     \
+    if ((__imm) == _CMP_NGE_US || (__imm) == _CMP_NGE_UQ)                      \
+      __ret = _mm_cmpnge_ss((__a), (__b));                                     \
+    if ((__imm) == _CMP_NGT_US || (__imm) == _CMP_NGT_UQ)                      \
+      __ret = _mm_cmpngt_ss((__a), (__b));                                     \
+    if ((__imm) == _CMP_FALSE_OQ || (__imm) == _CMP_FALSE_OS)                  \
+      __ret = _mm_move_ss((__a), _mm_setzero_ps());                            \
+    if ((__imm) == _CMP_GE_OS || (__imm) == _CMP_GE_OQ)                        \
+      __ret = _mm_cmpge_ss((__a), (__b));                                      \
+    if ((__imm) == _CMP_GT_OS || (__imm) == _CMP_GT_OQ)                        \
+      __ret = _mm_cmpgt_ss((__a), (__b));                                      \
+    if ((__imm) == _CMP_TRUE_UQ || (__imm) == _CMP_TRUE_US)                    \
+      __ret = _mm_move_ss((__a), (__m128)wasm_i8x16_splat(0xFF));              \
+    if ((__imm) == _CMP_NLE_US || (__imm) == _CMP_NLE_UQ)                      \
+      __ret = _mm_cmpnle_ss((__a), (__b));                                     \
+    __ret;                                                                     \
+  })
```

Comment:
The above #defines seem to have gotten rewritten. What happened there? Are these whitespace relayouted, or are there functional changes?

Reply:
relayout, the unchanged code is relative small, so the whole file of avxintrin.h and test_avx.cpp are formatted by clang-format, while only the changed lines of test_sse.h are formatted.